### ISSUE TITLE
Tracks can only be shown in all or none profiles despite toggle in "Configure map" (by profile)

### ISSUE THREAD

**Issue by MarioSwitch** [osmandapp/OsmAnd#19805]
### Description

In "Configure map", all changes are applied only to the selected profile. The "Tracks" toggle applies to all profiles, so you can't show tracks only for some profiles...
I set this issue as a bug report, despite all other settings concerns only the selected profile. 

### Steps to reproduce

Go to "Configure map" and enable/disable "Tracks", this change is applied to all profiles.

### Actual result

"Tracks" toggle in "Configure map" is synced between profiles, so you can't show tracks only in some profiles. Tracks can only be visible on all profiles, or invisible on all profiles.

### Expected result

You can choose to show/hide tracks for specific profiles (not for all), using "Tracks" toggle in "Configure map".

### Your Environment (required)

OsmAnd Version: 4.7.16
Android/iOS version: Android 14
Device model: Google Pixel 7
Crash-Logs: N/A

**Comment by yuriiurshuliak** [comment]
The bug has been reproduced.  To reproduce this issue, follow these steps: enable displaying one track on two profiles, then disable the display of this track on one profile and check the second profile.

As a result, the track visibility setting impacts all profiles. The "Tracks" toggle in the "Configure map" settings is synchronized across profiles, preventing the ability to display tracks selectively on certain profiles. 

OsmAnd~ 4.8.0#2270m, released: 2024-05-08


https://github.com/osmandapp/OsmAnd/assets/127092082/f86f6b91-66c2-43f3-bf51-cb19c5e91b72

**Comment by vshcherb** [comment]
Works as designed. We don't want specific Track selection for each profile

**Comment by MarioSwitch** [comment]
@vshcherb Yes, but in this case, why putting a toggle for that in the profile configuration? If this behaviour is expected, we could just have Tracks selection only in the track menu 🤔 Because, the weird thing is that all toggles in "Configure Map" concerns only the selected profile except this one...

**Comment by sonora** [comment]
> (...) Because, the weird thing is that all toggles in "Configure Map" concerns only the selected profile except this one...

Yes, but contrary to 'Configure profile' (which explicitly states it) there is no 'rule' that everything in 'Configure map' is profile specific.

Some configurations should rather be kept 'global' to avoid user confusion if things 'disappear', e.g. with an automated profile change like when starting a navigation. Whether the track visibility should fall under that remains to be seen, but historically we always had it as 'global', so it's not a recent bug or a regression.
